upstream-community-operators/storageos/storageos.v0.0.11.clusterserviceversion.yaml

```diff
@@ -0,0 +1,268 @@
+apiVersion: operators.coreos.com/v1alpha1
+kind: ClusterServiceVersion
+metadata:
+  name: storageosoperator.v0.0.11
+  namespace: placeholder
+  annotations:
```

Comment:
Additional annotations can be found here: https://github.com/operator-framework/community-operators/blob/master/docs/required-fields.md#required-fields-within-your-csv